GkeConnectGateway/tests/Unit/V1/Client/GatewayControlClientTest.php

```diff
@@ -77,8 +79,7 @@ public function generateCredentialsTest()
         $transport->addResponse($expectedResponse);
         // Mock request
         $name = 'name3373707';
-        $request = (new GenerateCredentialsRequest())
-            ->setName($name);
+        $request = (new GenerateCredentialsRequest())->setName($name);
```

Comment:
I personally prefer the previous version:
```php
$request = (new GenerateCredentialsRequest())
    ->setName($name);
```

My main reason is because of the closure with "()" for the new keyboard adds a bit of noise in my head so it "feels" better having the method one line down, but it is absolutely a nit.

Reply:
I agree! Although since these files are autogenerated, we'd need to make this change in `gapic-generator-php`.